src/tsconfig.json

```diff
@@ -12,5 +12,6 @@
     "sourceMap": true,
     "strict": true,
     "target": "es5"
-  }
+  },
+  "include": ["."]
```

Comment:
Just wondering what the reason for this `include` option is. I assume it's necessary as part of this change. Could you explain?

Reply:
I use the `include` option instead of providing `src` argument in `package.json` build command,
So I can use the `-p` option to use a different configuration for the `es2015` build.

Without this change you can't run `tsc -p src/tsconfig.es2015.json src`, which will result in

```
Option 'project' cannot be mixed with source files on a command line
```

Reply:
Of course, the value of this `include` option can be anyhting else to fit your needs, I included the whole folder but we can just set it to: `"include": ["./index.ts"]` or "include": ["**/*.ts"], actually we just wan to transpile `index.ts`

Reply:
Thanks for accommodating my laziness to spend the few minutes remembering this @eweap :)